processlib/activity.py

```diff
@@ -326,58 +349,43 @@ class StartViewActivity(StartMixin, ViewActivity):
 
 
 class EndActivity(Activity):
-    def instantiate(self, **kwargs):
-        super(EndActivity, self).instantiate(**kwargs)
-        self.start()
-        self.finish()
-
-    def finish(self, **kwargs):
-        super(EndActivity, self).finish(**kwargs)
-
-        update_fields = []
-        if not self.process.finished_at:
-            self.process.finished_at = self.instance.finished_at
-            update_fields.append("finished_at")
-
-        if not self.process.status == self.process.STATUS_DONE:
-            self.process.status = self.process.STATUS_DONE
-            update_fields.append("status")
-
-        self.process.save(update_fields=update_fields)
+    def __init__(self, redirect_url_callback=None, completed_message="", **kwargs):
```

Comment:
I think the old callback name should probably only be an alias for the `EndRedirectActivity`.
I also don't understand why we explicitly set the completed message again. It's the same as in the base class, right?

Reply:
Your right, with this I wanted to let users set individual messages on endactivity but its already provided via Parent